lib/plugins/aws/package/lib/addExportNameForOutputs.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  addExportNameForOutputs() {
+    const outputs = this.serverless.service.provider.compiledCloudFormationTemplate.Outputs;
+    Object.keys(outputs).forEach((key) => {
+      outputs[key] = Object.assign({}, outputs[key], {
+        Export: {
+          Name: `sls-${this.serverless.service.service}-${this.provider.getStage()}-${key}`,
+        },
+      });
```

Comment:
Instead of rewriting whole object, why not just assign name as:

```javascript
for (const [key, data] of Object.entries(outputs)) {
  data.Export = { Name: `sls-...` }
}
```

Additional question is, shouldn't we add it only if we detect that there's no `Export` property. What if there's some plugin which configures extra output with already configred `Export` property?

@pgrzesik what do you think?

Reply:
I think that is a very valid suggestion @medikoo. I didn't though about plugins potentially setting `Export` name, but I think it's highly likely that such tool exists - I agree that we should not set it if it's somehow already set :+1:

Reply:
that's a great point @medikoo ! I will add that check and also your suggestion is cleaner as well 👍